admin/includes/functions/admin_access.php

```diff
@@ -374,7 +374,7 @@ function zen_validate_user_login(string $admin_name, string $admin_pass): array
                 $response = zen_call_function(ZC_ADMIN_TWO_FACTOR_AUTHENTICATION_SERVICE, [$result['admin_id'], $result['admin_email'], $result['admin_name']]);
```

Comment:
This is where the enable/disable switch will go?  Can we also check for localhost or a private IP and skip if true?

Reply:
It's really hard to do that by relying on any IP-based conditions, because all servers have a localhost IP defined on themselves, and if they're on any network infrastructure other than directly on public internet, they'll be on a private IP range within their internal network behind the firewall.

And, then it would be super hard to test/verify the operation of MFA functionality in dev if it self-disabled.

So, no.